teammapper-backend/src/map/entities/mmpNode.entity.ts

```diff
@@ -96,4 +96,7 @@ export class MmpNode {
   @Column({ nullable: false })
   @Generated('increment')
     orderNumber: number;
+
+  @Column({ type: 'timestamptz', nullable: true, default: () => 'now()' })
```

Comment:
just fyi: this change will add a default date for all existing nodes as well (affected by the migration). Should we only add it for NEW nodes as a default or is the other case fine? Its not a big problem I guess but old maps will than stay active for maybe 2 months after this migration in the worst case (because otherwise they would have been deleted tomorrow and with this change you reactivated them for another 30 days)

Reply:
I think this is fine, since we don't know when the previous nodes were created. They could be a day old or 29 days.